### PR TITLE
adding cron event driver example and doc

### PR DESCRIPTION
**Pull request by tenczar**
Adds a cron driver to our event driver examples and adds documentation from using the driver.

### CODE REVIEW

File: docs/_guides/cron-event-driver.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Event Driver: cron
```
Comment (pzmrzy) [inline docs/_guides/cron-event-driver.md]: should add `layout: default` for jekyll

File: docs/_guides/cron-event-driver.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+The `cron` event driver fires events on a schedule determined by the cron expression passed to the event driver when created. This allows Dispatch to trigger functions in response to events generated on a scheduled basis.
+
+## Getting Started
+This guide assumes that you have a working Dispatch instance and access to the Dispatch CLI configured to work with your Dispatch instance. If you need helping creating a Dispatch instance or configuring your Dispatch CLI see the quickstart guide [here](../_front/quickstart.md)
+
+### Create a Function
+For this walkthrough we will use the hello.py example available [here](../../examples/python3). To begin we will seed Dispatch with the images we need for this function.
```
Comment (berndtj) [inline docs/_guides/cron-event-driver.md]: I don't think this link works.  The code isn't in the gh-pages branch that this is created from.  However, you should be able to link to examples/hello-world in the documentation

File: docs/_guides/cron-event-driver.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Event Driver: cron
+The `cron` event driver fires events on a schedule determined by the cron expression passed to the event driver when created. This allows Dispatch to trigger functions in response to events generated on a scheduled basis.
+
+## Getting Started
+This guide assumes that you have a working Dispatch instance and access to the Dispatch CLI configured to work with your Dispatch instance. If you need helping creating a Dispatch instance or configuring your Dispatch CLI see the quickstart guide [here](../_front/quickstart.md)
```
Comment (berndtj) [inline docs/_guides/cron-event-driver.md]: use the jekyl syntax for creating links (a.k.a. `[quickstart]({{ '/documentation/front/quickstart' | relative_url }})`)

File: docs/_guides/cron-event-driver.md (hunk 2 above)
Comment (berndtj) [reply docs/_guides/cron-event-driver.md]: Also use jekyl syntax

File: docs/_guides/cron-event-driver.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+### Create a Function
+For this walkthrough we will use the hello.py example available [here](../../examples/python3). To begin we will seed Dispatch with the images we need for this function.
+
+```bash
```
Comment (berndtj) [inline docs/_guides/cron-event-driver.md]: If they followed the quickstart, they already have the seed images.  You may want to state that they only need to run create-seed if the images haven't already been created.

File: docs/_guides/cron-event-driver.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    "services": null,
+    "status": "READY",
+    "tags": []
+}
+```
+
+### Create the Event Driver
+Next we need to create the event driver. This requires two steps. First we need to create the event driver type. We can do that with the following command. 
+```bash
+$ dispatch create eventdrivertype cron dispatchframework/cron-driver
```
Comment (berndtj) [inline docs/_guides/cron-event-driver.md]: We should probably explicitly specify a tag... likely not 'latest'

File: docs/_guides/cron-event-driver.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+Next we can create an instance of the event driver and configure it with our cron spec. Here you can replace the `5 * * * *` with any valid cron expression excluding the command to run. The event driver will handle firing an event for you.
+```bash
+$ dispatch create eventdriver cron --name cron-driver --set cron="5 * * * *"
```
Comment (berndtj) [inline docs/_guides/cron-event-driver.md]: Give it a more descriptive name like "every-5-seconds"

**Comment by tenczar** [comment]
Closing this pull request. Moved to standalone repo under dispatchframework. See here for latest https://github.com/dispatchframework/dispatch-events-cron/pull/1

**Pull request closed without merging**